app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -90,7 +90,7 @@ class BrowserToolbarView(
                 setToolbarBehavior()
 
                 elevation = resources.getDimension(R.dimen.browser_fragment_toolbar_elevation)
-
+                @Suppress("MagicNumber")
```

Comment:
Hm I don't see a magic number here? :). Is this suppress needed?

Reply:
Thanks for catching this, it wasn't meant to be part of the PR.